ansible-ipi-install/roles/installer/tasks/02_get_oc.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Get the ocp client tar gunzip file
+  get_url:
+    url: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/{{ version }}/openshift-client-linux-{{ version }}.tar.gz"
+    dest: "{{ extract_dir }}"
+    owner: "{{ myuser }}"
+    group: "{{ myuser }}"
+    mode: '0755'
+
+- name: "Untar the openshift-client-linux-{{ version }}.tar.gz"
```

Comment:
as brillantly suggested by @e-minguez , we want to redownload oc client later on to match the release we target, to avoid potential issues if "latest" oc is not so latest (i think it's 4.2 since a couple of days)

Reply:
I just realized that the code here wouldn't work for "latest" but only if we specified a specific version. This is because the naming scheme for latest is something like openshift-client-linux-4.4.0-0.nightly-2020-01-24-141203.tar.gz and the version was set to "latest" you'd get nothing back. Will fix logic here.